### PR TITLE
updated layout for entity compact card (mdcompact)

### PR DESCRIPTION
**Pull request by victor-xm**

### CODE REVIEW

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss [hunk 1]
```diff
@@ -17,47 +30,105 @@
...
+}
+
+ .card-icon img{
+    width:         70px;
+     height:        70px;
+    overflow:      hidden;
+    border-radius: 3px;
+}
+.card-custom .card-icon.avatar{
+padding:0px;
```
Comment (andrewkikot) [inline src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:52]: please stick to the general code styling

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss [hunk 2]
```diff
@@ -1,3 +1,16 @@
+.description{
+color: #909090;
```
Comment (andrewkikot) [inline src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:2]: please stick to the general code styling

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss [hunk 3]
```diff
@@ -17,47 +30,105 @@
...
     &.has-avatar {
         .card-title {
-            margin-left: 160px;
+            margin-left: 60px;
         }
-
-        .avatar-container-card {
-            margin-top: -45px;
+        .card-desc{
+           // margin-left: 67px; -- no need with current avatar position
```
Comment (andrewkikot) [inline src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:82]: do not know what exactly position we talking about, but if you sure it is displayed correctly, then remove commented code please

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss [hunk 4]
```diff
@@ -17,47 +30,105 @@
...
         }
+
+      .avatar-position{
+      position:absolute;
```
Comment (andrewkikot) [inline src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:86]: please stick to the general code styling

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss [hunk 5]
```diff
@@ -17,47 +30,105 @@
...
+      margin-top: 6px;
+      font-size: 18px;
+    }
+
+
+}
+
+
+.function-list{
+width:100%;
```
Comment (andrewkikot) [inline src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:114]: please stick to the general code styling

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss [hunk 6]
```diff
@@ -68,16 +139,16 @@
...
-        max-height: 1000px;
+        max-height: 250px;
 
         .card-desc-toggle {
             transform: rotate(225deg);
         }
     }
 
-    &-inner {
+/*    &-inner {
```
Comment (andrewkikot) [inline src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:149]: please remove commented code if it is unnecessary

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss [hunk 7]
```diff
@@ -115,7 +186,7 @@
             }
 
             .container-fluid {
-                padding-left: 0 !important;
+//                padding-left: 0 !important;
```
Comment (andrewkikot) [inline src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:189]: please remove commented code if it is unnecessary

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.ts [hunk 8]
```diff
@@ -10,24 +10,34 @@ import { EntityCardComponent } from '../entity-card/entity-card.component';
...
 
     public jsfAttributes: any;
     public showLoader: boolean;
     public isDescFull: boolean;
 
+    // transferred from entity-detail-fab
+    public showEditButton: boolean;
+    public showEditOptions: boolean = false;
+    public showEditSubOptions: boolean = false;
+endsWith
```
Comment (andrewkikot) [inline src/app/xm-entity/entity-card-compact/entity-card-compact.component.ts:40]: not really sure it's even compiling with this line of code

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss [hunk 9]
```diff
@@ -17,47 +30,105 @@
...
+
+.row-compact{
+  margin-left:0px;
+  margin-right:0px;
+}
+
+.card-custom-schema-compact{
+border-top: solid 1px #E0E0E0;
+margin-top: 5px !important;
+padding-top: 7px;
```
Comment (andrewkikot) [inline src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:71]: please stick to the general code styling

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.ts [hunk 10]
```diff
@@ -10,24 +10,34 @@ import { EntityCardComponent } from '../entity-card/entity-card.component';
 import { RatingListSectionComponent } from '../rating-list-section/rating-list-section.component';
 import { XmEntityService } from '../shared/xm-entity.service';
 
+// transferred from entity-detail-fab
```
Comment (andrewkikot) [inline src/app/xm-entity/entity-card-compact/entity-card-compact.component.ts:13]: why this kind of comments in master branch?

Comment (andrewkikot) [reply src/app/xm-entity/entity-card-compact/entity-card-compact.component.ts:13]: if its shares similar logic and behavior maybe it's reasonable to move in common helper or something

File: src/app/xm-entity/function-list-section-compact/function-list-section-compact.component.scss [hunk 11]
```diff
@@ -1,11 +1,21 @@
-.limetedWidth {
+.limitedWidth {
     max-width: 160px;
 
     @media screen and (max-width: 420px) {
         max-width: 100%;
     }
 }
 
+div.displayblock
```
Comment (andrewkikot) [inline src/app/xm-entity/function-list-section-compact/function-list-section-compact.component.scss:9]: please stick to the general code styling

File: src/app/xm-entity/function-list-section-compact/function-list-section-compact.component.scss [hunk 12]
```diff
@@ -14,8 +24,19 @@
...
         display: block;
     }
 }
+
+.row-compact{
+  margin-left:0px;
+  margin-right:0px;
+}
+
+.container-fluid{
```
Comment (andrewkikot) [inline src/app/xm-entity/function-list-section-compact/function-list-section-compact.component.scss:40]: do we really need this empty block here?

File: src/app/xm-entity/function-list-section-compact/function-list-section-compact.component.html [hunk 13]
```diff
@@ -12,59 +12,55 @@
       <mat-tab *ngFor="let f of customFunctions; let i = index;"
                [label]="f.name | i18nName: principal"
                [tabIndex]="f.key === 'AREA' ? 'AREA': i">
+
+<!--VK: obsolete
```
Comment (andrewkikot) [inline src/app/xm-entity/function-list-section-compact/function-list-section-compact.component.html:16]: can not understand, why leave commented code, especially in this strange manner (<!--VK: obsolete)

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss (hunk 1 above)
Comment (victor-xm) [reply src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:52]: done

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss (hunk 2 above)
Comment (victor-xm) [reply src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:2]: done

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss (hunk 3 above)
Comment (victor-xm) [reply src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:82]: done

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss (hunk 4 above)
Comment (victor-xm) [reply src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:86]: done

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss (hunk 5 above)
Comment (victor-xm) [reply src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:114]: done

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss (hunk 9 above)
Comment (victor-xm) [reply src/app/xm-entity/entity-card-compact/entity-card-compact.component.scss:71]: done

File: src/app/xm-entity/entity-card-compact/entity-card-compact.component.ts (hunk 10 above)
Comment (victor-xm) [reply src/app/xm-entity/entity-card-compact/entity-card-compact.component.ts:13]: the first backward-compatible step to getting away from a fab button on the entity detail screen, causing temporary code duplication to edit entity name/description. Code marked for better understanding.

File: src/app/xm-entity/function-list-section-compact/function-list-section-compact.component.scss (hunk 11 above)
Comment (victor-xm) [reply src/app/xm-entity/function-list-section-compact/function-list-section-compact.component.scss:9]: done

File: src/app/xm-entity/function-list-section-compact/function-list-section-compact.component.scss (hunk 12 above)
Comment (victor-xm) [reply src/app/xm-entity/function-list-section-compact/function-list-section-compact.component.scss:40]: removed

File: src/app/xm-entity/function-list-section-compact/function-list-section-compact.component.html (hunk 13 above)
Comment (victor-xm) [reply src/app/xm-entity/function-list-section-compact/function-list-section-compact.component.html:16]: comment changed

**Review by major13ua — APPROVED** [review]

**Review by andrewkikot — APPROVED** [review]

**Pull request merged**